tests/test_people_page.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from pages.PeoplePage import PeoplePage
+from tests.locators_people_page import PeoplePageLocator, URLLocators
+
+
+class TestPagePeople:
+    def test_people_title_001(self):
+        driver = PeoplePage(self.driver)
+        driver.go_to_page(URLLocators.URL_PEOPLE)
+        print(driver.get_title())
```

Comment:
no prints in repository code.please remove

Reply:
Sorry, I don't understand what wrong?

Reply:
line 11
it's not a good practice to leave a print function in your code

Reply:
Ok